org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/FindWindow.java

```diff
@@ -1228,7 +1229,7 @@ private void setValues(Listbox listColumn, Listbox listOperator, String[] fields
             	{
                 	listColumn.setSelectedItem(li);
             		selected = true;
-            		break;
```

Comment:
why the need to remove the break ?

Reply:
The list of columns was being shown incomplete.